man/rmd-fragments/navbar-configuration.Rmd

```diff
@@ -75,3 +76,27 @@ navbar:
       href: http://twitter.com/hadleywickham
       aria-label: Twitter
 ```
+
+You can also customize the "brand" part of the navbar which appears on the left and is not collapsed on any device.
```

Comment:
Do you think we could do:

```yaml
navbar:
  structure:
    left:  [intro, reference, articles, tutorials, news]
    right: [search, github]
    brand: [package, version]
```

Or is brand too different in structure to left and right?
```

Reply:
I think it's ok to simplify a bit. Otherwise really we'd need to change the structure to:

```yaml
navbar:
  links:
    structure: ...
    components: ...
  brand:
    structure: ...
    components: ...
```

and that seems like overkill given that most people won't customize the brand.

Either way, this section should probably get it's own heading, and mention that the components are just pasted together.

Reply:
I haven't changed the structure of the YAML for now, is it fine?